git/cmd.py

```diff
@@ -13,6 +13,8 @@
 import errno
 import mmap
 
+from collections import OrderedDict
```

Comment:
for future reference: this issue got fixed in #431, but other incompatibilities with Py2.6 remain, e.g. https://travis-ci.org/gitpython-developers/GitPython/jobs/133041770

I won't be spending time on those (for now), I'll just stick with the latest GitPython 1.x on Py2.6 (cfr. https://github.com/hpcugent/easybuild-framework/pull/1781)